requirements.txt

```diff
@@ -29,7 +29,8 @@ sphinxcontrib-redoc==1.6.0
 texttable==1.6.3
 tornado==6.1
 asyncpg==0.21.0
-pytest-postgresql==2.5.2
+pytest-postgresql==2.5.3
```

Comment:
This one will be part of the `inmanta-dev-dependencies`, no?

Reply:
yes, but core contains many deps that are also in dev deps. Do we clean this up?

Reply:
Indeed. And core also doesn't make a distinction between `requirements.txt` and `requirements.dev.txt`. I think we should clean this up at some point, but we don't have to do that now.